ra-optimisation/greedy-optimisation/src/main/java/com/farao_community/farao/greedy_optimisation/GreedyOptimisation.java

```diff
@@ -0,0 +1,23 @@
+/**
```

Comment:
Replace /** by /* in copyright header, because these are not Javadoc comments (I know, we have this issue everywhere in FARAO source)